### PR TITLE
Store all images locally

### PR DESCRIPTION
**Pull request by jacob-g**
Fixes issue #90

### CODE REVIEW

File: resources/css/main.css [hunk 1]
```diff
@@ -454,7 +454,7 @@ dl dt {
     position: absolute;
     box-shadow: none;
     background-color: transparent;
-    background-image: url("https://scratch.mit.edu/images/nav-search-glass.png");
+    background-image: url("../nav-search-glass.png");
```
Comment (Kenny2github) [inline resources/css/main.css:457]:
```suggestion
    /* @embed */
    background-image: url("../nav-search-glass.png");
```

**Review by ahmetlii — APPROVED** [review]

File: resources/main.css [hunk 2]
```diff
@@ -372,7 +372,7 @@ dl dt {
 }
 
 #navigation .logo a {
-    background-image: url("https://scratch.mit.edu/images/logo_sm.png");
+    background-image: url("Scratch-logo-sm.png");
```
Comment (Kenny2github) [inline resources/main.css:375]:
```suggestion
    /* @embed */
    background-image: url("Scratch-logo-sm.png");
```

File: resources/main.css [hunk 3]
```diff
@@ -457,7 +457,7 @@ dl dt {
     position: absolute;
     box-shadow: none;
     background-color: transparent;
-    background-image: url("https://scratch.mit.edu/images/nav-search-glass.png");
+    background-image: url("nav-search-glass.png");
```
Comment (Kenny2github) [inline resources/main.css:460]:
```suggestion
    /* @embed */
    background-image: url("nav-search-glass.png");
```

**Review by Kenny2github — CHANGES_REQUESTED** [review]
Might as well make use of ResourceLoader

**Review by Kenny2github — APPROVED** [review]

**Pull request merged**